extensions/3DTILES_metadata/1.0.0/README.md

```diff
@@ -0,0 +1,1623 @@
+              },
+              "countries": {
+                "description": "The countries that this tile overlaps",
+                "type": "ARRAY",
+                "componentType": "STRING"
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "root": {
+    "refine": "ADD",
+    "geometricError": 19266.834945,
+    "boundingVolume": {
+      "region": [-2.356194490192345, 0.0, -1.5707963267948966, 0.7853981633974483, -2119.1943359375, 5615.173828125]
+    },
+    "content": {
+      "uri": "terrain.b3dm"
+    },
+    "extensions": {
+      "3DTILES_metadata": {
+        "class": "tile",
+        "properties": {
+          "horizonOcclusionPoint": [-0.4142135640377178, -1.0000000040187549, 0.44683832670325535],
+          "countries": ["United States", "Canada", "Mexico"]
+        }
+      }
+    }
+  }
+}
+```
+
+#### Implicit Tile Metadata
+
+When using the [`3DTILES_implicit_tiling` extension](../../3DTILES_implicit_tiling/0.0.0) tile metadata is stored in subtree buffers. Tile metadata only exists for available tiles and is tightly packed by increasing tile index. See [Implicit Tiling Availability](../../3DTILES_implicit_tiling/0.0.0#availability) for details about tile ordering. To access individual tile metadata, implementations may create a mapping from tile indices to tile metadata indices.
+
+Below is an example subtree JSON:
+
+```jsonc
+{
+  "buffers": [
+    {
+      "byteLength": 99692
+    }
+  ],
+  "bufferViews": [
+    {
+      "buffer": 0,
+      "byteLength": 688,
+      "byteOffset": 0
+    },
+    {
+      "buffer": 0,
+      "byteLength": 688,
+      "byteOffset": 688
+    },
+    {
+      "buffer": 0,
+      "byteLength": 2048,
+      "byteOffset": 1376
+    },
+    {
+      "buffer": 0,
+      "byteLength": 49152,
+      "byteOffset": 3424
+    },
+    {
+      "buffer": 0,
+      "byteLength": 24576,
+      "byteOffset": 50528
+    },
+    {
+      "buffer": 0,
+      "byteLength": 8196,
+      "byteOffset": 75104
+    },
+    {
+      "buffer": 0,
+      "byteLength": 16388,
+      "byteOffset": 83304
+    }
+  ],
+  "tileAvailability": {
+    "bufferView": 0
+  },
+  "contentAvailability": {
+    "bufferView": 1
+  },
+  "childSubtreeAvailability": {
+    "bufferView": 2
+  },
+  "extensions": {
+    "3DTILES_metadata": {
+      "class": "tile",
+      "properties": {
+        "horizonOcclusionPoint": {
+          "bufferView": 3
+        },
+        "countries": {
+          "bufferView": 4,
+          "arrayOffsetBufferView": 5,
+          "stringOffsetBufferView": 6
+        }
+      }
+    }
+  }
+}
+```
+
+### Group Metadata
+
+Metadata may be assigned to groups. Groups represent collections of contents. Contents are assigned to groups with the `3DTILES_metadata` content extension.
+
+Group metadata can be paired with [`3DTILES_multiple_contents`](../../3DTILES_multiple_contents/0.0.0) to assign multiple contents in a tile to different groups.
```

Comment:
Not sure that the novice reader is ready for this detail yet, but it would be welcomed in an Appendix on how other extensions interact with metadata.